src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/finalparameters/Example3.java

```diff
@@ -0,0 +1,21 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="FinalParameters">
+      <property name="ignorePrimitiveTypes" value="true"/>
+    </module>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.finalparameters;
+
+// xdoc section -- start
+public class Example3 {
+  public Example3() { }
+  public Example3(final int m) { }
+  public Example3(final int m,int n) { }
```

Comment:
```suggestion
  public Example3(final int m ,int n) { }
```

Reply:
@nrmancuso done

Reply:
I think you mean  `public Example3(final int m , int n) { }` not  `public Example3(final int m  ,int n) { }` and this is done 
please correct me if i am wrong